L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -1047,6 +1047,9 @@ namespace trklet {
     double stripLength_2S_{5.0250};
 
     //Following values are used for duplicate removal
+    //The bin edges for rinv were chosen using the output tracks without duplicate removal for a large number of events
```

Comment:
This text is hard to understand. e.g. In "track at that index", it's unclear what the word "index2 refers to.